File: hsStudentProfile/web/src/app/models/Student.ts
import { IStudent } from './IStudent';

export class Student implements IStudent {
	id: number;
    profilename: string;
    firstname: string;
    lastname: string;
    primaryemail: string;
    highschoolid: number;
    graduationyear: number;
    additionalinfo: string;
    gpa: number;
    sattestdate: Date;
    satscore: number;
    acttestdate: Date;
    actscore: number;
    ncaaid: string;
    phone: number;
    streetaddress1: string;
    streetaddress2: string;
    city: string;
    state: string;
    zip: string;
    displayaddrandphone: number;
    profilepictureid: number;
    height: string;
    weight: string;
    collegemajor: string;
}

File: hsStudentProfile/webServices/src/webServices/Entities/Tables/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace webServices.Entities
{
    public class Student : IEntityBase
    {
        public Student()
        {
        }
        public int id { get; set; }
        public int? highschoolid { get; set; }
        public int? graduationyear { get; set; }
        public string additionalinfo { get; set; }
        public decimal? gpa { get; set; }
        public DateTime? sattestdate { get; set; }
        public int? satscore { get; set; }
        public DateTime? acttestdate { get; set; }
        public int? actscore { get; set; }
        public string ncaaid { get; set; }
        public int? profilepictureid { get; set; }
        public string height { get; set; }
        public string weight { get; set; }
        public DateTime? created { get; set; }
        public string collegemajor { get; set; }
    }
}
